packages/caravan-psbt/src/psbtv0/psbt.test.ts

```diff
@@ -103,9 +103,15 @@ describe("translatePsbt", () => {
   const TRANSACTIONS = TEST_FIXTURES.transactions;
 
   const tx = _.cloneDeep(TEST_FIXTURES.transactions[0]);
+  tx.inputs.forEach((input) => {
+    input.sequence = 0xffffffff;
+  });
   const ms = MULTISIGS[0];
   it("handles P2WSH transactions", () => {
     const fixture = TEST_FIXTURES.transactions.find((tx) => tx.segwit);
+    fixture.inputs.forEach((input) => {
+      input.sequence = 0xffffffff;
+    });
```

Comment:
i didn't want to change the caravan-bitcoin fixtures to just support tests for sequence, so I just piggyback on the existing fixture.